README.rst

```diff
@@ -71,8 +74,11 @@ if you do so, then we'll be able to include it here and spread the word about it
 Installation
 -------------
 
-coala-bears can be installed with ``pip3 install coala-bears``. If you need more
-information about the installation and dependencies, take a look at our
+In order to run a coala-bear you need to install Python. It is recommended, that you install
+Python3 >= 3.3 from `here <https://www.python.org/downloads/>`__.
```

Comment:
maybe drop it. have something like: ``To install coala-bears, you first need Python 3 or higher installed. If you do not have it, it is recommended to install Python >= 3.3 from here (give link).``